src/libraries/System.Net.WebProxy/src/System/Net/WebProxy.cs

```diff
@@ -95,8 +95,7 @@ public bool UseDefaultCredentials
         }
 
         private static Uri? CreateProxyUri(string? address) =>
-            address == null ? null :
-            address.IndexOf("://", StringComparison.Ordinal) == -1 ? new Uri("http://" + address) :
+            address == null ? null : !address.Contains("://") ? new Uri("http://" + address) :
```

Comment:
Did you change this formatting or did the auto-fixer?  If the latter, looks like we need to fix the fixer's handling of trivia.  Regardless, can you please add back the newline before `!address`?

Reply:
The auto-fix did. Good catch on the trivia. I fixed it in the analyzer :)